mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -98,7 +99,8 @@ def Rock_Conv2DBwdWeightOp : Rock_Conv2DOpBase<"conv2d_bwd_weight">
 {
   dag additionalArgs = (ins Optional<TensorOrMemRefRankOf<[F32], [5]>>:$workspace,
                             OptionalAttr<IndexAttr>:$kBlocks,
-                            I32Attr:$numCu);
+                            I32Attr:$numCu,
+                            OptionalAttr<I32Attr>:$utilityBlockSize);
```

Comment:
Why are we doing this instead of setting `derivedBlockSize`?

Reply:
This is yet another block size only for the utility kernel. I want to make it sure they are not confused.

Reply:
I suppose ... we definitely need a ticket to define separate ops for the utility kernels

Reply:
> I suppose ... we definitely need a ticket to define separate ops for the utility kernels

https://github.com/ROCmSoftwarePlatform/llvm-project-private/issues/730